Dispatchfile

```diff
@@ -1,39 +1,54 @@
-         steps = [
+         inputs=["src-git"],
+         steps=[
              v1.Container(
                  name="fetch-master",
-                 image = "mesosphere/dispatch-dind:v0.5.2",
+                 image="mesosphere/dispatch-dind:v0.5.2",
                  workingDir="/workspace/src-git",
                  args=["git", "fetch", "origin", "master"]),
 
              v1.Container(
-                 name = "dispatch-integration-test",
-                 image = "mesosphere/kubeaddons-ci:dispatch@sha256:80a6f3c8775133b4ad2fab3d44ab4fbb4b541dbc03fcdb296b13f67611427d8c",
-                 command = ["make","dispatch-test"],
-                 workingDir = "/workspace/src-git",
-                 env = [k8s.corev1.EnvVar(name = "DISPATCH_CI", value = "true"),
-		        k8s.corev1.EnvVar(name = "AWS_REGION", value= "us-west-2"),
-                        k8s.corev1.EnvVar(name = "AWS_ACCESS_KEY_ID",
-                                          valueFrom = secretVar("d2iq-dispatch-aws-creds",
-                                                                "AWS_ACCESS_KEY_ID")),
-                        k8s.corev1.EnvVar(name = "AWS_SECRET_ACCESS_KEY",
-                                          valueFrom = secretVar("d2iq-dispatch-aws-creds",
-                                                                "AWS_SECRET_ACCESS_KEY")),
-                        k8s.corev1.EnvVar(name = "SSH_KEY_BASE64",
-                                          valueFrom = secretVar("d2iq-dispatch-git-ssh-base64",
-                                                                "ssh-privatekey-base64"))
-			])
-		])
+                 name="dispatch-integration-test",
+                 image="mesosphere/kubeaddons-ci:dispatch@sha256:80a6f3c8775133b4ad2fab3d44ab4fbb4b541dbc03fcdb296b13f67611427d8c",
+                 command=["make", "dispatch-test"],
+                 workingDir="/workspace/src-git",
+                 env=[k8s.corev1.EnvVar(name="DISPATCH_CI", value="true"),
+                      k8s.corev1.EnvVar(name="AWS_REGION", value="us-west-2"),
+                      k8s.corev1.EnvVar(name="AWS_ACCESS_KEY_ID",
+                                        valueFrom=secretVar("d2iq-dispatch-aws-creds",
+                                                            "AWS_ACCESS_KEY_ID")),
+                      k8s.corev1.EnvVar(name="AWS_SECRET_ACCESS_KEY",
+                                        valueFrom=secretVar("d2iq-dispatch-aws-creds",
+                                                            "AWS_SECRET_ACCESS_KEY")),
+                      k8s.corev1.EnvVar(name="SSH_KEY_BASE64",
+                                        valueFrom=secretVar("d2iq-dispatch-git-ssh-base64",
+                                                            "ssh-privatekey-base64"))
+                      ],
+                 resources=k8s.corev1.ResourceRequirements(
+                     requests={
+                         "cpu": k8s.resource_quantity("2000m"),
```

Comment:
2 cpus for a request to run remote tests seem quite high don't they? I would reccomend that we remove the requests and increase limits so that they can burst to the appropriate level.

Reply:
We want this to schedule to a node that has room. These requests will ensure the pod is scheduled to a node that should satisfy the needs. If we went with defaults, it would still be possible (maybe even likely) for the application to be OOM killed without reaching the limits if the node itself runs out of memory.

Since these are not permanent allocations, I'm not concerned with the size of the requests at this time. If it turns out that these pods are not getting scheduled due to there being a lack of nodes available to satisfy the requests, we could come back at this at that time.